src/applications/Applications.ExtractImages/ExtractImagesCliOptions.cs

```diff
@@ -13,7 +13,7 @@ public class ExtractImagesCliOptions : CliOptions
         // Required
 
         [Option(shortName: 'p', longName: "project-id", Required = true, HelpText = "The project identifier")]
-        public string ProjectId { get; set; } = null!;
+        public string? ProjectId { get; set; } = null;
```

Comment:
Should this not be the same as `CohortCsvFile` below it? I don't think this will ever be null due to `Required = true` ?

Reply:
Hopefully :)